src/App/Exercises/HtmlCss/ArrayRendering/Data.js

```diff
@@ -0,0 +1,38 @@
+export const Data = [
```

Comment:
Widzę, że nazwałaś zmienną i plik z dużej litery. Istnieje taka niepisana zasada, że z dużej litery nazywamy komponenty reactowe, a resztę rozpoczynamy od małej literki. Nie jest to żaden straszny problem, tylko taka mała pomoc dla współtwórców projektu: od razu widać, co jest normalną zmienną czy funkcją, a co jest komponentem 😃

Reply:
zaraz to poprawie, taki odruch chyba że nowy plik nazwałam z dużej i potem już tak poszło 😅